go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/abcxyz/pkg v1.0.4
 	github.com/google/go-cmp v0.6.0
 	github.com/google/renameio v1.0.1
+	github.com/google/uuid v1.6.0
```

Comment:
I remain very skeptical that we need UUIDs for anything 😄

Reply:
Oh, wonder how that got back in there, thought I had run tidy